core/lib/utils/Cargo.toml

```diff
@@ -26,6 +26,7 @@ hex = "0.4"
 reqwest = { version = "0.11", features = ["blocking"] }
 itertools = "0.10.5"
 metrics = "0.21"
+sqlx = "0.7.3"
```

Comment:
This crate may be used by something that doesn't need DAL, so I would really love to not depend on `sqlx` here, since we have tracked history of starting to used whatever we can use uncontrollably.
This can become a huge PITA in the future (look at `zksync_types` 🥲 ).

Reply:
Maybe we can move functions that use SQLx types to the DAL crate, since most likely they are used in the context where DAL is available?

Reply:
Actually not needed, removed.